packages/jest-types/src/Config.ts

```diff
@@ -322,7 +322,7 @@ export type ProjectConfig = {
   dependencyExtractor?: string;
   detectLeaks: boolean;
   detectOpenHandles: boolean;
-  displayName?: DisplayName;
+  displayName?: string | DisplayName;
```

Comment:
this is wrong, as `normalize` will... normalize a string from `InitialOptions` into the object form before creating `ProjectConfig`. `projects` might need to be `Partial<InitialOptions>` instead?

This is just from reading the diff in this PR, I haven't opened the project try to figure out the correct approach

Reply:
> `projects` might need to be `Partial<InitialOptions>` instead?

Based on your description, that may be. Though I got `ProjectConfig` directly from the docs: https://jestjs.io/docs/en/configuration#projects-arraystring--projectconfig. It's possible the docs need to be updated too.

<img width="809" alt="Screen Shot 2021-02-25 at 8 34 26 AM" src="https://user-images.githubusercontent.com/3850064/109168313-4e4bbd00-7744-11eb-86e7-6aa63e1d00c0.png">